mtprotoproxy.py

```diff
@@ -119,10 +120,22 @@ def decrypt(self, data):
 KEY_LEN = 32
 IV_LEN = 16
 HANDSHAKE_LEN = 64
-MAGIC_VAL_POS = 56
+PROTO_TAG_POS = 56
 DC_IDX_POS = 60
 
-MAGIC_VAL_TO_CHECK = b'\xef\xef\xef\xef'
+PROTO_TAG_ABRIDGED = b'\xef\xef\xef\xef'
+PROTO_TAG_INTERMEDIATE = b'\xee\xee\xee\xee'
+
+class RpcFlags(enum.Flag):
+    NONE                  =        0x0
+    NOT_ENCRYPTED         =        0x2
+    HAS_AD_TAG            =        0x8
+    MAGIC                 =     0x1000
+    EXTMODE2              =    0x20000
```

Comment:
Also no idea what this flag does. There are also EXTMODEs 1 and 3, 1 is never used. /shrug